didx-xyz/yoma-mobile
# YOMA-532 - feat(Challenges): store challenges in state

## Scope [YOMA-532](https://yomaworld.atlassian.net/browse/YOMA-532)
- store challenges in state

![](https://media.giphy.com/media/wNbMrlmfatKqA/giphy.gif)
## Work Done
- created reducer and middleware to handle the challenges state
- adding middleware to fetch and normalise and then set the challenges data in state.

## Steps to Test
- run tests
- run the app - look in the debugger - make sure the challenges data is stored in state
